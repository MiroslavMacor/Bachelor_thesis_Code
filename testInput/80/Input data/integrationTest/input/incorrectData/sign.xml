<?xml version="1.0"?><remsig><person><userId>1236</userId></person><certificateId>7</certificateId><password>1234asd56</password><data>abcdefghijklmn</data></remsig>
