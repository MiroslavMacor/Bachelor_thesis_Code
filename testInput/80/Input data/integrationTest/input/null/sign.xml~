<?xml version="1.0"?><remsig><person><userId>1236</userId></person><certificateId>7</certificateId><password>123456</password><data></data></remsig>
