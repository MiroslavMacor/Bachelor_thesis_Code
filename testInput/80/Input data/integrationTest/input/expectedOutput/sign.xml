<?xml version="1.0" encoding="UTF-8"?><remsig>  <returnType>OK</returnType>  <signature>tlLV5m4E5FV9m7icpejmVQLi0Yj/Ka0KHlQ0SNhUs6I+sBupmGlDQ990mORfbBeU5kQDSGPNPD7YQHZeAop6459KTP8Ml3aYWkP0l2dlxr4oB9x1lZ3QI++sFvEGv9YMaYdonN96nAhPpJOGyGQyfoONpITmrgLhqj0r4qj1snkcplaxIUHZe1SMvF71z8eP8++RIrBzZNM2wJoMbPl6vmYTjkpJ4BI4oJau0gbHZ0KSNQqc7lAKqK2B+iw7DMVwfSh42wMuBIC5kqG2sCoizuwM1Hj/WafPYzH6QmzZ9VeiAOA0BdYdBCB7d8rZMOI6yV+EpzxJDLqxjbuLWEpGP6sF7LVYnLWoIqaaz1GLmfOf1brHQxFs+iVD1BeNWCq9KNpOgQpn1dMtRJXP2ynH/HqpqsZzIEPmgI3cnDZNtdL48tAUVWgHAEJ596bZLVLB688vp02GuTSNWud+CXa32w4XUvfjtjrpc7/d/38K2E0ZdaLhw/x9XzdB17JegDvGdzlyo7Gc6Ti8NfCvb7JGajuyRtrlydj8lx9YbU2BmXujnQaHN37OE8BYUwv2p7GQNXbdgXt02j/C82JMTe1mmTDcH2a1mERRjabWLYiEWxUCgAXfscOkgi3/HOBDH2rankT6oEC1I/+2DSMvI/uGWEo+Ywwku6RSEnpfNiEyfJk=</signature>  <operationId>1463326314920</operationId></remsig>
